src/mixxxmainwindow.cpp

```diff
@@ -1073,8 +1073,11 @@ bool MixxxMainWindow::loadConfiguredSkin() {
 
 bool MixxxMainWindow::eventFilter(QObject* obj, QEvent* event) {
     if (event->type() == QEvent::ToolTip) {
-        // return true for no tool tips
-        switch (m_toolTipsCfg) {
+        // always show tooltips in the preferences window
+        QWidget* actWin = QApplication::activeWindow();
+        if (actWin && QLatin1String(actWin->metaObject()->className()) != "DlgPreferences") {
```

Comment:
that could also be
``` c++
if (actWin && actWin->objectName() != "DlgPreferencesDlg")
```
both work, I don't know what's preferred. Also, I didn't manage to cast char to QString, the attempts were rather elaborate for a string comparison.